Sources/DolbyIORTSCore/StreamOrchestrator.swift

```diff
@@ -264,6 +264,22 @@ private extension StreamOrchestrator {
         let (_, success) = await (startSubscribeStateUpdate, startSubscribe)
         return success
     }
+    
+    func stopAudio(for sourceId: String) async {
+        switch await stateMachine.currentState {
+        case let .subscribed(state):
+            if let source = state.sources.first (where: { $0.sourceId.value == sourceId }),
+               source.isPlayingAudio == true {
+                await withTaskGroup(of: Void.self) { [weak self] group in
```

Comment:
we don't need a task group if its only one task to execute:

we can just call:
`self.subscriptionManager.unprojectAudio(for: source)`

instead of:

```
await withTaskGroup(of: Void.self) { [weak self] group in
guard let self = self else { return }
                    group.addTask {
                        self.subscriptionManager.unprojectAudio(for: source)
                    }
                }
```

Reply:
So no risk other tasks can be done concurrently, right? If so, I will remove it.

Reply:
No, `await withTaskGroup` is for awaiting and performing a set concurrent tasks before executing the subsequent lines of code. Here since we have only one task to perform on the `subscriptionManager` - so there is no concurrency.